kvbc/cmf/categorized_kvbc_msgs.cmf

```diff
@@ -201,6 +201,7 @@ Msg BatchedInternalNodeKey 5002 {
     # The version of this key is the *tree* version, not block version
     uint64 version
     NibblePath path
+    string address
```

Comment:
OK so we serialize all Merkle tree in the same collection, and we distinguish the merkle tree node keys by the account address ?

Reply:
Yes, we can have all MPT-s (the Global state, which will have an empty address field and each account's tree) in the same storage space (CF).